webapp/lib/controller.dart

```diff
@@ -128,25 +143,50 @@ void command(UIAction action, Data actionData) {
       ProjectConfigurationData projectConfigurationData = actionData;
       saveProjectConfiguration(projectConfigurationData.config);
       break;
+    case UIAction.saveProjectConfiguration:
+      savePackageConfiguration();
+      break;
+  }
+}
+
+enum NavAction {
+  ALLPROJECTS,
+  DASHBOARD
+}
+
+void _toggleNavActions(NavAction navAction, bool show) {
+  switch (navAction) {
+    case NavAction.ALLPROJECTS:
+      view.navView.allProjectsLink.classes.toggle('nav-links__link--show', show);
```

Comment:
The controller shouldn't access HTML elements directly and change classes on them... please create a wrapper method on the navView object that does this instead

Reply:
created a wrapper method in view